### PR TITLE
Refresh network-cyberoam-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-cyberoam-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 1]
```diff
@@ -2,48 +2,574 @@
 id: network-cyberoam-snmp
 title: Cyberoam
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-## Prerequisites
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 2]
```diff
@@ -2,48 +2,574 @@
...
+| Services | Net-Cyberoam-Services-SNMP-custom | Contrôle l'état des services                        |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Cyberoam-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias          | Modèle de service                       | Description                                               | Découverte |
+|:---------------|:----------------------------------------|:----------------------------------------------------------|:----------:|
+| Traffic-Global | Net-Cyberoam-Traffic-Global-SNMP-custom | Contrôle le traffic réseau de plusieurs interfaces réseau | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:43]:
```suggestion
| Traffic-Global | Net-Cyberoam-Traffic-Global-SNMP-custom | Contrôle le trafic réseau de plusieurs interfaces réseau | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 3]
```diff
@@ -2,48 +2,574 @@
...
+|:----------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Cyberoam-SNMP-custom** |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                | Description                                                             |
+|:-------------------------------|:------------------------------------------------------------------------|
+| Net-Cyberoam-SNMP-Traffic-Name | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:66]:
```suggestion
| Net-Cyberoam-SNMP-Traffic-Name | Découvre les interfaces réseau et supervise leur statut et leur utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 4]
```diff
@@ -2,48 +2,574 @@
...
+	--mode=interfaces \
+	--hostname=10.0.0.1 \
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--interface='.*' \
+	--name \
+	--add-status \
+	--add-traffic \
+	--critical-status='' \
+	--warning-in-traffic='80' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:350]: is that a percentage?

File: pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 5]
```diff
@@ -2,48 +2,573 @@
...
+* **Net-Cyberoam-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Cyberoam-SNMP-custom" label="Net-Cyberoam-SNMP-custom">
+
+| Service Alias | Service Template                  | Service Description                                  |
+|:--------------|:----------------------------------|:-----------------------------------------------------|
+| Cpu           | Net-Cyberoam-Cpu-SNMP-custom      | Check the rate of utilization of CPU for the machine |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:29]:
```suggestion
| Cpu           | Net-Cyberoam-Cpu-SNMP-custom      | Check the rate of CPU utilization for the machine |
```

File: pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 6]
```diff
@@ -2,48 +2,573 @@
...
+	--mode=interfaces \
+	--hostname=10.0.0.1 \
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--interface='.*' \
+	--name \
+	--add-status \
+	--add-traffic \
+	--critical-status='' \
+	--warning-in-traffic='80' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:349]: is this a percentage?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 7]
```diff
@@ -2,48 +2,573 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=54590b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=67975b/s;80;90;; 'interface_name1#interface.traffic.out.bitspersecond'=86710b/s;80;90;; 'interface_name2#interface.traffic.out.bitspersecond'=80594b/s;80;90;;
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:359]:
```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=20b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=20b/s;80;90;; 'interface_name1#interface.traffic.out.bitspersecond'=20b/s;80;90;; 'interface_name2#interface.traffic.out.bitspersecond'=20b/s;80;90;;
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md [hunk 8]
```diff
@@ -2,48 +2,574 @@
...
+	--critical-in-traffic='90' \
+	--warning-out-traffic='80' \
+	--critical-out-traffic='90' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=54590b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=67975b/s;80;90;; 'interface_name1#interface.traffic.out.bitspersecond'=86710b/s;80;90;; 'interface_name2#interface.traffic.out.bitspersecond'=80594b/s;80;90;;
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cyberoam-snmp.md:360]:
```suggestion
OK: All interfaces are ok | 'interface_name1#interface.traffic.in.bitspersecond'=20b/s;80;90;; 'interface_name2#interface.traffic.in.bitspersecond'=20b/s;80;90;; 'interface_name1#interface.traffic.out.bitspersecond'=20b/s;80;90;; 'interface_name2#interface.traffic.out.bitspersecond'=20b/s;80;90;;
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**